### PR TITLE
Add CuboidRegion example usage

### PR DESCRIPTION
**Pull request by anonyco**
Snippet was written by Pluggg: https://discord.com/channels/268444645527126017/347745984446726144/999942207664554024

### CODE REVIEW

File: worldedit-core/src/main/java/com/sk89q/worldedit/regions/CuboidRegion.java [hunk 1]
```diff
@@ -34,6 +34,29 @@
 
 /**
  * An axis-aligned cuboid. It can be defined using two corners of the cuboid.
+ *
+ * Example usage:
+ * <pre>
+ * <code>
+ * Bukkit.getScheduler().runTaskTimerAsynchronously(this, () -> {
```
Comment (me4502) [inline worldedit-core/src/main/java/com/sk89q/worldedit/regions/CuboidRegion.java:41]: Why are you using platform-specific code in the example usage here, let alone async code which is not safe for alterations to the world

File: worldedit-core/src/main/java/com/sk89q/worldedit/regions/CuboidRegion.java [hunk 2]
```diff
@@ -34,6 +34,29 @@
...
+ *    for (final Pair<Cuboid, List<WantedMaterial>> mine : mines.values()) {
```
Comment (me4502) [inline worldedit-core/src/main/java/com/sk89q/worldedit/regions/CuboidRegion.java:42]: These classes don't exist, that's bad for an example

File: worldedit-core/src/main/java/com/sk89q/worldedit/regions/CuboidRegion.java [hunk 3]
```diff
@@ -34,6 +34,29 @@
...
+ *        final World world = BukkitAdapter.adapt(mine.getKey().getWorld());
```
Comment (me4502) [inline worldedit-core/src/main/java/com/sk89q/worldedit/regions/CuboidRegion.java:43]: Again, platform-specific

File: worldedit-core/src/main/java/com/sk89q/worldedit/regions/CuboidRegion.java [hunk 4]
```diff
@@ -34,6 +34,29 @@
...
+ *
+ *        final CuboidRegion selection = new CuboidRegion(
```
Comment (me4502) [inline worldedit-core/src/main/java/com/sk89q/worldedit/regions/CuboidRegion.java:45]: This example usage doesn't actually explain anything beyond the constructor. Given these classes don't exist, it's not useful

File: worldedit-core/src/main/java/com/sk89q/worldedit/regions/CuboidRegion.java [hunk 5]
```diff
@@ -34,6 +34,29 @@
...
+ *            world,
+ *            toBlockVector(mine.getKey().getLowerNE()),
+ *            toBlockVector(mine.getKey().getUpperSW())
+ *        );
+ *
+ *        try (final EditSession session = WorldEdit.getInstance().newEditSessionBuilder().world(world).build()) {
```
Comment (me4502) [inline worldedit-core/src/main/java/com/sk89q/worldedit/regions/CuboidRegion.java:51]: This is more showing how to perform a random pattern edit, which is much better covered by the docs than this

File: worldedit-core/src/main/java/com/sk89q/worldedit/regions/CuboidRegion.java [hunk 6]
```diff
@@ -34,6 +34,29 @@
...
+ *            RandomPattern pattern = toRandomPattern(mine.getValue());
+ *            session.setBlocks((Region) selection, pattern);
+ *        }
+ *
+ *    }
+ * },  0, 10 * 60 * 20);
```
Comment (me4502) [inline worldedit-core/src/main/java/com/sk89q/worldedit/regions/CuboidRegion.java:57]: Repeating tasks here massively over complicate an example

**Review by me4502 — COMMENTED** [review]
I've written out a lot of issues with this PR, but tbh I do not see a need for it in the first place. Unless you can show why the constructor and other docs for this class aren't enough, an example doesn't seem necessary. Especially an example that doesn't explain anything, and spends all its time over-complicating surrounding irrelevant code

**Comment by anonyco** [comment]
A few comments:

1. Thank you for your review of this commit.

2. I thank you for and apologize for all the time you put into writing the thankful comments about the code.

3. I opened this PR for another person (Pluggg, the author of the snippet.)

I myself tend to trust when people say they found a snippet/explanation useful to them even though I don't see the use for myself, and I respect and will honor the WorldEdit policy on this. As this snippet will never live up to the WorldEdit standards without a complete rewrite and I am not invested enough in seeing this through, I am respectfully closing this PR with special appreciation to you for your time and attention to this.

**Pull request closed without merging**